rusk/src/lib/http/rusk.rs

```diff
@@ -35,6 +35,7 @@ impl HandleRequest for Rusk {
             ("driver", Some(_), _) => true,
             ("contract_owner", Some(_), _) => true,
             ("upload_driver", Some(_), _) => true,
```

Comment:
This needs to be removed if you don't use it anymore.

I'll handle it in the next PR (the one for the error codes)